package DB연결;

public class 에러처리 {

	public static void main(String[] args) throws Exception{
		//try {
			int x= 10/0;
		//} catch (Exception e) {
			System.out.println("산술연산부분에서 에러남.");
		//}
		
		
		System.out.println("내가 실행 될까요??");
	}

}
